### DISCUSSION TITLE
GlobalExcludePaths

### DISCUSSION THREAD

**Discussion by bhc993** [community/community#64636] [Programming Help]
### Select Topic Area

Question

### Body

Hello, using the Powershell script "GlobalExcludePaths" to update Protection Policies. Doses this script not allow uppercase and lower case of the same exclude? Seems it only adds the one it reads first.
Example:

*.dmp
*.DMP

Are these mutually exclusive?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @bhc993 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
